src/raster/r.series.diversity/r.series.diversity.py

```diff
@@ -115,86 +115,83 @@
 # % required: -r,-s,-h,-e,-p,-g,-n
 # %end
 
+# %option G_OPT_M_NPROCS
+# %end
+
+# %option
+# % key: memory
+# % type: integer
+# % required: no
+# % multiple: no
+# % label: Maximum memory to be used (in MB)
+# % description: Maximum memory to be used (in MB)
+# % answer: 300
+# %end
```

Comment:
Please only define changes here with regard to the already defined options:

https://github.com/OSGeo/grass/blob/ff2da11b41e6943ff3d83f4801df203228b81ecd/lib/gis/parser_standard_options.c#L757

Reply:
Hi @neteler, I completely overlooked the standard `G_OPT_MEMORYMB` option. Thanks. I'll change that.